libs/blocks/table/table.js

```diff
@@ -97,6 +97,36 @@ function handleHeading(table, headingCols) {
   });
 }
 
+function handleAddOnContent(table) {
+  const addOnKey = 'ADDON';
+  const addOns = [...table.querySelectorAll('.section-row-title')]
+    .filter((row) => row.innerText.includes(addOnKey));
+  if (addOns.length) {
+    table.classList.add('has-addon');
+    addOns.forEach((addOn) => {
+      const addOnRow = addOn.parentElement;
+      addOnRow.remove();
+      const [position, order, style] = addOn.innerText.split('_')
+        .filter((key) => key !== addOnKey).map((key) => key.toLowerCase());
+      if (position && order) {
+        [...table.querySelector('.row-heading').children].forEach((headCol) => {
+          const indexAttr = 'data-col-index';
+          const colIndex = headCol.getAttribute(indexAttr);
+          if (colIndex > 1) { // ignore the key column
+            const content = `${position}-${order}`;
+            const tag = createTag('div', { class: content }, [...addOnRow.children]
+              .find((col) => col.getAttribute(indexAttr) === colIndex)?.innerHTML);
+            if (style) tag.classList.add(style);
```

Comment:
I worry a bit about the fact that these classes can be quite generic, like `label`, which might already have some styles attached from a more general stylesheet. Should we make them a bit more specific to the use-case at hand by programatically prepending a more specific string?

Reply:
I am prepending `addon-` to the tag now!